internal/metrics/metricsManager.go

```diff
@@ -91,17 +94,17 @@ type API interface {
 	ClusterValidationChanged(clusterVersion string, emailDomain string, clusterValidationType models.ClusterValidationID)
 	InstallationStarted(clusterVersion string, clusterID strfmt.UUID, emailDomain string, userManagedNetworking string)
 	ClusterHostInstallationCount(clusterID strfmt.UUID, emailDomain string, hostCount int, clusterVersion string)
-	ClusterHostsNTPFailures(clusterID strfmt.UUID, emailDomain string, hostNTPFailureCount int)
```

Comment:
why did you remove this method?

Reply:
It is not used anywhere. I consulted the original author and he conceded too that it should be removed because it is not required any more